testdata/baselines/reference/submoduleAccepted/conformance/plainJSGrammarErrors.errors.txt.diff

```diff
@@ -1,69 +1,33 @@
-@@= skipped -4, +4 lines =@@
- plainJSGrammarErrors.js(17,9): error TS18041: A 'return' statement cannot be used inside a class static block.
- plainJSGrammarErrors.js(20,5): error TS1089: 'static' modifier cannot appear on a constructor declaration.
- plainJSGrammarErrors.js(21,5): error TS1089: 'async' modifier cannot appear on a constructor declaration.
--plainJSGrammarErrors.js(22,5): error TS8009: The 'const' modifier can only be used in TypeScript files.
- plainJSGrammarErrors.js(22,11): error TS1248: A class member cannot have the 'const' keyword.
--plainJSGrammarErrors.js(23,5): error TS8009: The 'const' modifier can only be used in TypeScript files.
- plainJSGrammarErrors.js(23,11): error TS1248: A class member cannot have the 'const' keyword.
+@@= skipped -11, +11 lines =@@
  plainJSGrammarErrors.js(26,11): error TS1030: 'async' modifier already seen.
  plainJSGrammarErrors.js(28,11): error TS1029: 'static' modifier must precede 'async' modifier.
  plainJSGrammarErrors.js(29,5): error TS1031: 'export' modifier cannot appear on class elements of this kind.
 -plainJSGrammarErrors.js(29,5): error TS8009: The 'export' modifier can only be used in TypeScript files.
  plainJSGrammarErrors.js(30,5): error TS1031: 'export' modifier cannot appear on class elements of this kind.
++plainJSGrammarErrors.js(34,9): error TS8017: Signature declarations can only be used in TypeScript files.
  plainJSGrammarErrors.js(34,22): error TS1005: '{' expected.
  plainJSGrammarErrors.js(35,9): error TS1054: A 'get' accessor cannot have parameters.
-@@= skipped -24, +21 lines =@@
- plainJSGrammarErrors.js(54,8): error TS1030: 'export' modifier already seen.
- plainJSGrammarErrors.js(55,8): error TS1044: 'static' modifier cannot appear on a module or namespace element.
- plainJSGrammarErrors.js(56,22): error TS1090: 'static' modifier cannot appear on a parameter.
--plainJSGrammarErrors.js(56,22): error TS8012: Parameter modifiers can only be used in TypeScript files.
- plainJSGrammarErrors.js(57,7): error TS1029: 'export' modifier must precede 'async' modifier.
- plainJSGrammarErrors.js(58,26): error TS1090: 'export' modifier cannot appear on a parameter.
--plainJSGrammarErrors.js(58,26): error TS8012: Parameter modifiers can only be used in TypeScript files.
- plainJSGrammarErrors.js(59,25): error TS1090: 'async' modifier cannot appear on a parameter.
--plainJSGrammarErrors.js(59,25): error TS8012: Parameter modifiers can only be used in TypeScript files.
+ plainJSGrammarErrors.js(36,9): error TS1049: A 'set' accessor must have exactly one parameter.
+@@= skipped -26, +26 lines =@@
  plainJSGrammarErrors.js(60,7): error TS1030: 'async' modifier already seen.
  plainJSGrammarErrors.js(61,1): error TS1042: 'async' modifier cannot be used here.
  plainJSGrammarErrors.js(62,5): error TS1042: 'async' modifier cannot be used here.
 -plainJSGrammarErrors.js(62,5): error TS8009: The 'async' modifier can only be used in TypeScript files.
```

Comment:
This error seems to still be missing? But, probably "good" because the old error was duplicated for some reason.

Reply:
Another case of duplicate checks.